src/Component/Style/Style.tsx

```diff
@@ -79,6 +79,7 @@ export interface StyleProps extends Partial<StyleDefaultProps> {
   colorRamps?: {
     [name: string]: string[]
   };
+  useBrewerColorRamps?: boolean;
```

Comment:
Should be moved to `StyleDefaultProps` and added with true to `defaultProps`.

Reply:
IMHO a default value of `true` does not really make sense, since setting it to `true` means that you also need to configure custom color ramps (which by default are not set).

Reply:
Ok so i didn't get it. I'll read it again.

Reply:
Thought `Style` was the final component... Just forget it and merge. :)